runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingConfig.java

```diff
@@ -150,6 +150,17 @@ default boolean showMinnowOverlay()
 
 	@ConfigItem(
 		position = 10,
+		keyName = "flyingFish",
```

Comment:
I'll commit this via GitHub's UI
```suggestion
		keyName = "flyingFishNotification",
```